index.js

```diff
@@ -37,6 +39,11 @@ function shell(commands, options) {
       var context = _.extend({file: file}, options.templateData)
       command = gutil.template(command, context)
 
+      if (options.verbose) {
+        var title = util.format('>> %s%s', gutil.colors.blue(command), '\n')
+        process.stdout.write(title)
```

Comment:
Why not use [`gutil.log`](https://github.com/gulpjs/gulp-util#logmsg) to print the command?